starter-kits/nextjs-starter-kit/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "description": "This repository contains a collaborative starter kit using Liveblocks and Next.js.",
+  "license": "Apache-2.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint --fix",
+    "typecheck": "tsc --project tsconfig.json --noEmit"
+  },
+  "dependencies": {
+    "@liveblocks/client": "^0.19.0",
+    "@liveblocks/node": "^0.19.0",
+    "@liveblocks/react": "^0.19.0",
+    "@radix-ui/react-dialog": "^1.0.2",
+    "@radix-ui/react-popover": "^1.0.2",
+    "@radix-ui/react-select": "^1.1.1",
+    "@radix-ui/react-tabs": "^1.0.1",
+    "@radix-ui/react-tooltip": "^1.0.2",
+    "base-64": "^1.0.0",
+    "clsx": "^1.2.1",
```

Comment:
I've spent time thinking about this too! I agree that string interpolation would be best for examples, and it makes for better copy-and-pasting.

However, I'd also argue that we're trying to provide a repo full of industry best practices, so that no copy-and-pasting is needed—we want people to take this repo and make it their own. I can't see many Next.js users building a large project _without_ using a library like `clsx` (it's used about 30 times just in the starter). I'm always happy see a package like this used in a starter kit, even if it's not the one I normally use, because string interpolation becomes quite messy with logic.

`clsx` is quite popular:

```
NPM Weekly Downloads
---
  React  18,246,887
   clsx   9,229,120
Next.js   3,571,339
```

What do you think? I still don't mind changing it if you think it's best 🙂

Reply:
I've quickly talked about this with @stevenfabre and we've decided it's okay merge this with `clsx` added, so I can get working on the installer 🙂